src/main/java/com/comphenix/protocol/wrappers/ComponentParser.java

```diff
@@ -51,7 +51,7 @@ public static Object deserialize(Object gson, Class<?> component, StringReader s
 	// Should only be needed on 1.8.
 	private static Object deserializeLegacy(Object gson, Class<?> component, StringReader str) {
 		try {
-			if(readerConstructor == null){
+			if (readerConstructor == null){

```

Comment:
```suggestion
			if (readerConstructor == null) {

```

Reply:
Lol where did you find these? I literally CTRL + SHIFT + F `if(`

Reply:
I just look at the github diff :)